fs_error_test.go

```diff
@@ -81,6 +81,25 @@ func setupErrorRepos(t *testing.T) (*sql.Context, CleanupFunc) {
 	return ctx, cleanup
 }
 
+func brokenFS(
+	brokenType brokenType,
+	fs billy.Filesystem,
+) (billy.Filesystem, error) {
+	dotFS, err := fs.Chroot(".git")
+	if err != nil {
+		return nil, err
+	}
+
+	var brokenFS billy.Filesystem
+	if brokenType == 0 {
```

Comment:
what's _0_? do we need a const for this?

Reply:
It means no broken flags, I've just added a constant.